server/setup_postgresql.sh

```diff
@@ -1,40 +1,45 @@
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/027-alertsessions.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/028-addrespondedby.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/029-alterrespondedphonenumberstobeanarray.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/030-removeoldbuttonid.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/031-addclientlanguage.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/032-addbuttonsvitalscachetable.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/033-addgatewaysvitalscachetable.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/034-addclientsextensiontable.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/035-addsentgatewayvitalsalert.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/036-addbuttonsvitalsalerts.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/037-addmorestatuses.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/038-addbuttonsvitals-snrrssi.sql
-sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER -v "ON_ERROR_STOP=1" --set=sslmode=require -f ./db/039-remove-hubs.sql
+#!/bin/bash
+
+if [[ -z "${PG_PASSWORD}" ]]; then
+  pg_password=$1
+else
+  pg_password="${PG_PASSWORD}"
+fi
+
+if [[ -z "${PG_USER}" ]]; then
+  pg_user=$2
+else
+  pg_user="${PG_USER}"
+fi
+
+if [[ -z "${PG_HOST}" ]]; then
+  pg_host=$3
+else
+  pg_host="${PG_HOST}"
+fi
+
+if [[ -z "${PG_PORT}" ]]; then
+  pg_port=$4
+else
+  pg_port="${PG_PORT}"
+fi
+
+if [[ -z "${PG_USER}" ]]; then
+  pg_dbname=$5
+else
+  pg_dbname="${PG_USER}"
+fi
+
+echo "Running PostgreSQL commands..."
+
+# Create the required PostgreSQL extensions
+sudo PGPASSWORD=$pg_password psql -U $pg_user -h $pg_host -p $pg_port -d $pg_dbname -v "ON_ERROR_STOP=1" --set=sslmode=require -c 'CREATE EXTENSION IF NOT EXISTS "pgcrypto"'
```

Comment:
I actually don't think you need this. It's already done in `setup_postgresql_local_dev.sh`.

Reply:
Thank you for the feedback, I tried to run the test again after removing this part of the code, and I observed it is necessary to keep it.